code/modules/mob/dead/observer/spells.dm

```diff
@@ -26,6 +26,13 @@ var/global/list/boo_phrases=list(
 	range = 1 // Or maybe 3?
 
 /obj/effect/proc_holder/spell/aoe_turf/boo/cast(list/targets)
+
+	// First check if there are shadowlings nearby, because flickering lights will damage them.
+	var/shadowling_present = 0
+	for(var/mob/living/carbon/C in range(7, usr)) // (Range is Ghost Boo range (1) + Shadowling Veil range (5) + 1 for good measure)
+		if (istype(C,/datum/species/shadow/ling))
```

Comment:
This check won't work; a carbon can't be a species datum.

You'd want to cycle through humans in range and you'd want to check with `get_species`